### DISCUSSION TITLE
Optional Purchase Option

### DISCUSSION THREAD

**Discussion by riotrah** [adirh3/Fluent-Search#515] [Ideas]
This is bonkers - the quality, insane iteration rate, responsivity etc put this in the league of paid commercial products (that are worth their price tag). 

I'm sure the thought has crossed your mind, but i just wanted to add my vote that I'd love to pay for this and have perhaps some purchase option beyond a donation link. Eg the Windows Store version could be paid whilst the ones on GH remain free.

In any case, this is one of my favorite OSS projects ever, and I thank you for your tremendous effort. At the very least, I hope this project continues to remain enjoyable for you intrinsically. Thanks for working on this.

**Comment by adirh3** [comment]
Thanks @riotrah, glad to hear you liked Fluent Search.
I have thought about adding paid themes, but I really don't want to have any kind of features behind a pay wall.

Making an optional paid version is an excellent idea, the only issue is that the store version is problematic due to the Appx (container) format, so I'd like to not encourage users to install it through the store. 
Hopefully, I will be able to solve most Appx issues and then go with your suggestion.

Also, please note that Fluent Search is technically not OSS project, even though the community is incredibly involved the code is not open sourced.
